src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String startUp = " Hello! I'm Duke\n"
+                + " What can I do for you?";
+        ArrayList<Task> list = new ArrayList<>();
```

Comment:
Perhaps a better name for this list could be taskList, or tasks, to make it clearer as to what the list contains